devicetypes/vision-raytseng/vision-4-in-1-motion-sensor.src/vision-4-in-1-motion-sensor.groovy

```diff
@@ -0,0 +1,495 @@
+/**
+ *  Vision 4-in-1 Motion Sensor
+ *
+ *  Author: Ray Tseng
+ */
+metadata {
+	definition (name: "Vision 4-in-1 Motion Sensor", namespace: "vision-raytseng", author: "Ray Tseng", vid: "generic-motion-8", ocfDeviceType: "x.com.st.d.sensor.motion") {
+		capability "Battery"
+		capability "Motion Sensor"
+		capability "Relative Humidity Measurement"
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+		capability "Tamper Alert"
+		capability "Health Check"
+		
+		fingerprint mfr:"0109", prod:"2021", model:"2112", deviceJoinName: "Vision Multipurpose Sensor" // zw:Ss2a type:0701 mfr:0109 prod:2021 model:2112 ver:32.32 zwv:7.13 lib:03 cc:5E,22,98,9F,6C,55 sec:85,59,80,70,5A,7A,87,8E,72,71,73,31,86,84
```

Comment:
Please add "Raw description" (literally), so it would look:
...deviceJoinName: "Vision Multipurpose Sensor" // Raw description: zw:Ss2a type...

Reply:
Okay, I have add "Raw description" comment in fingerprint.